src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocBlockTagLocationCheck.java

```diff
@@ -0,0 +1,199 @@
+////////////////////////////////////////////////////////////////////////////////
+// checkstyle: Checks Java source code for adherence to a set of rules.
+// Copyright (C) 2001-2019 the original author or authors.
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+////////////////////////////////////////////////////////////////////////////////
+
+package com.puppycrawl.tools.checkstyle.checks.javadoc;
+
+import java.util.Arrays;
+import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+import com.puppycrawl.tools.checkstyle.StatelessCheck;
+import com.puppycrawl.tools.checkstyle.api.DetailNode;
+import com.puppycrawl.tools.checkstyle.api.JavadocTokenTypes;
+
+/**
+ * <p>
+ * Checks that a
+ * <a href="https://docs.oracle.com/en/java/javase/11/docs/specs/doc-comment-spec.html#block-tags">
+ * javadoc block tag</a> appears only at the beginning of a line, ignoring leading asterisks and
+ * white space.
+ * </p>
+ * <p>
+ * Rationale: all javadoc block tags not placed at the beginning of a line are
+ * treated as the plain text. This may be intentional, but changing the text may
```

Comment:
Lets come come origin of Check. Lets change to something like this:
`Rationale: according to <a link>spec</a> all javadoc block tags should be placed at the beginning of a line, tags that placed at not the beginning  are treated as the plain text. To recognize intentional tag placement to text area it is better to escape @ symbol, and all non-escaped tags should be located at the beginiing of line. See NOTE section for details on howto escape.`

Reply:
done